### PR TITLE
Exercise 2

### PR DESCRIPTION
**Pull request by uyaramm**
This PR writes a program that asks the user for a number and prints the sum of the numbers 1 to n. However, only multiples of three or five are considered in the sum, e.g. 3, 5, 6, 9, 10, 12, 15 for n=17.

### CODE REVIEW

File: exercise_2.cpp [hunk 1]
```diff
@@ -0,0 +1,59 @@
+
+#include <iostream>
+using namespace std;
+
+/*Write a program that asks the user for a number n and prints the sum of the
+numbers 1 to n. However, only multiples of three or five are considered in the
+sum, e.g. 3, 5, 6, 9, 10, 12, 15 for n=17*/
+
+int getSum(int number) {
+  int i;
```
Comment (tmontovaneli) [inline exercise_2.cpp]: This variable can be declared in the for statement. Please consider.

File: exercise_2.cpp [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  case 2:
+    int number, sum;
+    cout << "Type a number and I'll give you the sum of of three or five: "
+         << endl;
+    cin >> number;
+
+    sum = getSum(number);
+    cout << "The sum of the multiple of three and five is: " << sum << endl;
+    ;
```
Comment (tmontovaneli) [inline exercise_2.cpp]: Useless semicolon.

File: exercise_2.cpp [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+void executeUserChoice(int choice) {
+
+  switch (choice) {
+  case 1:
+    executeTest();
+    break;
+
+  case 2:
```
Comment (tmontovaneli) [inline exercise_2.cpp:43]: Consider moving this logic, `case 2`, to a method that handles only it.

Comment (uyaramm) [reply exercise_2.cpp:43]: `sumWithUserInput` method was added.

**Review by tmontovaneli — APPROVED** [review]
Nice job! 🎉

**Pull request merged**